src/app/search/components/patient_stage/patient_stage.html

```diff
@@ -40,25 +40,25 @@ <h2 class="modal-title" id="titleLabel">Patient Staging</h2>
           <td rowspan="{{ location.results.length }}" href ng-click="vm.selectAll(location)">{{ location.location.name }}</td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected" ng-bind-html="vm.displayNames(location.results[0].patientRecordName)"></td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].gender.description }}</td>
-          <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].dateOfBirth | ehealthdate }}</td>
+          <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ vm.convertDobString(location.results[0].dateOfBirth) }}</td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].ssn }}</td>
-          <td>
+          <td align="center">
```

Comment:
align="center" isn't proper HTML5, but the CSS solutions are complicated and messy, so probably not worth dealing with right now